The compiled code failed because the variable name was not changed

The compiled code returns the following error:
error: not found: value initPath
[ERROR]  logInfo(s"Intel Optane PMem initialized with path: ${initPath}, size: ${pmemInitialSize} ")

--- a/src/main/scala/org/apache/spark/storage/BlockManager.scala
+++ b/src/main/scala/org/apache/spark/storage/BlockManager.scala
@@ -190,7 +190,7 @@ private[spark] class BlockManager(
   val numNum = conf.getInt("spark.yarn.numa.num", 2)
 
   if (pmemMode.equals("AppDirect")) {
-    if (!isDriver && pmemInitialPaths.size > 1) {
+    if (!isDriver && pmemInitialPaths.size >= 1) {
       if (numaNodeId == -1) {
         numaNodeId = executorId.toInt
       }

--- a/src/main/scala/org/apache/spark/storage/BlockManager.scala
+++ b/src/main/scala/org/apache/spark/storage/BlockManager.scala
@@ -210,8 +210,8 @@ private[spark] class BlockManager(
       }
 
       require(file.isDirectory(), "PMem directory is required for initialization")
-      PersistentMemoryPlatform.initialize(path_postfix, pmemInitialSize, 0)
-      logInfo(s"Intel Optane PMem initialized with path: ${path_postfix}, size: ${pmemInitialSize} ")
+      PersistentMemoryPlatform.initialize(file.getAbsolutePath, pmemInitialSize, 0)
+      logInfo(s"Intel Optane PMem initialized with path: ${file.getAbsolutePath}, size: ${pmemInitialSize} ")
     }
   } else if (pmemMode.equals("KMemDax")) {
     if (!isDriver) {

--- a/src/main/scala/org/apache/spark/storage/BlockManager.scala
+++ b/src/main/scala/org/apache/spark/storage/BlockManager.scala
@@ -210,8 +210,8 @@ private[spark] class BlockManager(
       }
 
       require(file.isDirectory(), "PMem directory is required for initialization")
-      PersistentMemoryPlatform.initialize(initPath, pmemInitialSize, 0)
-      logInfo(s"Intel Optane PMem initialized with path: ${initPath}, size: ${pmemInitialSize} ")
+      PersistentMemoryPlatform.initialize(path_postfix, pmemInitialSize, 0)
+      logInfo(s"Intel Optane PMem initialized with path: ${path_postfix}, size: ${pmemInitialSize} ")
     }
   } else if (pmemMode.equals("KMemDax")) {
     if (!isDriver) {

